pkg/process/config/yaml_config.go

```diff
@@ -164,6 +164,12 @@ func (a *AgentConfig) loadSysProbeYamlConfig(path string) error {
 		a.EnabledChecks = append(a.EnabledChecks, "TCP queue length")
 	}
 
+	if config.Datadog.GetBool(key(spNS, "process_tracer.enabled")) {
+		log.Info("system_probe_config.process_tracer.enabled detected, will enable system-probe with process_tracer")
```

Comment:
This is not a useful log line in my opinion. Doesn't the system-probe already log what modules are started?

Reply:
Removed.